.gitignore

```diff
@@ -19,6 +19,8 @@ examples/Tasks/main
 examples/Tuples/main
 examples/EncodeDecode/main
 examples/SafeMath/main
+examples/ElmWithRoc/webserver
+examples/ElmWithRoc/frontend/elm-stuff/*
```

Comment:
I think we should call this `ElmWebApp` as not all roc users may be familiar with Elm the front end language.

Reply:
I previously changed this to `ElmFrontendRocBackend`, what do you think of that?

Reply:
I think I'd prefer the shorter one I suggested, though they are both ok. It's not super important, I was just thinking about clarifying that this was a web application and using elm. Saying frontend and backend really only makes sense to someone familiar with web domain.